R/vulnerability/disasters-emergencies/flooding/2022-interim/england/service-availability/flooding-exposure-emergency-services.R

```diff
@@ -0,0 +1,103 @@
+library(dplyr)
+library(readr)
+library(osmdata)
+library(sf)
+library(geographr)
+
+flood_risk_oa <- read_rds("data/vulnerability/disasters-emergencies/flooding/flood-risk-output-areas.rds")
+
+# ---- Load Output Area boundaries ----
+# Source: https://geoportal.statistics.gov.uk/datasets/ons::output-areas-december-2011-boundaries-ew-bfe/about
+# Opening straight from the web takes far too long, so I manually downloaded a local copy and will load from that
+# boundaries_oa <- read_sf("https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/Output_Areas_December_2011_Boundaries_EW_BFE/FeatureServer/0/query?where=1%3D1&outFields=OA11CD&outSR=4326&f=json")
+boundaries_oa <- read_sf("data/vulnerability/disasters-emergencies/flooding/Output_Areas__December_2011__Boundaries_EW_BFE.shp")
+
+boundaries_oa <- st_transform(boundaries_oa, crs = 4326)
+
+# ---- Get map points for each type of emergency service ----
+# Bounding box for England
+england_bb <- getbb("England")
+
+fire <- 
+  opq(england_bb, timeout = 1000) %>%
+  add_osm_feature("amenity", "fire_station") %>%
+  osmdata_sf()
+
+police <- 
+  opq(england_bb, timeout = 1000) %>%
+  add_osm_feature("amenity", "police") %>%
+  osmdata_sf()
+
+ambo <- 
+  opq(england_bb, timeout = 1000) %>%
+  add_osm_feature("emergency", "ambulance_station") %>%
+  osmdata_sf()
+
+emergency_services <- 
+  bind_rows(
+    fire$osm_points,
+    police$osm_points,
+    ambo$osm_points
+  ) %>% 
+  select(osm_id, name)
+
+emergency_services <- st_transform(emergency_services, crs = 27700)
+
+# ---- Get Output Areas for each emergency service ----
+st_crs(emergency_services) <- 4326  # Need to do this to make the next block of code work
```

Comment:
Get warning on this. Think need to use `st_transform` to reproject data otherwise risk it doesn't re-project correctly.

Reply:
We're not reprojecting in this step - it's a bug-fix recommended by https://stackoverflow.com/a/62268361 because the full CRS isn't set when downloading from OpenStreetMap. Will make that clear in the code.

Reply:
Ah apologies thanks for clarifying. It might be due to different systems/versions of sf/osmdata but if I keep `boundaries_oa` at it's original crs 'OSGB36' and then set the `emergency_services` object to that same crs using `st_transform` I don't seem to get an error for the OSM data CRS or the join.